### ISSUE TITLE
[IL] Revenue calculations change with every page refresh

### ISSUE THREAD

**Issue by pie-for-providers-admin** [pieforproviders/pieforproviders#2862]
## 👉🏼 Steps to Reproduce

1. Go to staging
2. Log in with test@test credentials
3. Go to dashboard
4. Toggle between July * June 2022
5. See the revenue calculations change every time you toggle in the roll-up stats.
6. Note that the Potential & Max revenue also changes in the table.

## 🇺🇸 States this bug is happening in
- [ ] ALL
- [ ] Nebraska
- [X] Illinois

## 📸 Screenshots

Video below:

https://user-images.githubusercontent.com/78763415/182884472-d5ac501d-4221-4eb8-8c34-d3c34b8ab1da.mov

## 💻 Browser Info

Please complete the following information if accessing the app from a computer:

- Operating System: [e.g. iOS]
- Browser [e.g. Chrome, Safari]

**Comment by rebeccakarasiktw** [comment]
Hey @OliverCai0, @JonErvin and I were QAing this and we got the following errors when we tried to upload the spreadsheet. Let us know if we are doing the command wrong or if there's another issue to look into.
This is the command we used: heroku run bundle exec rails read_illinois_onboarding_cases --app pie-staging

![Image](https://user-images.githubusercontent.com/26929739/190266103-588144c8-0aba-4e68-b0c1-f2ea97a01c41.png)

**Comment by OliverCai0** [comment]
@rebeccakarasiktw @JonErvin 

**Updates**

I added the user in the example file to the database (password is testpass1234! and email is ildaycare1@test.com).  I also added the license types for all Businesses: 

1. day_care_center
2. license_exempt_day_care_center
3. group_child_care_homes
4. child_care_homes
5. license_exempt_day_care_home
6. in_home

As long the business license type is imported in a format similar to that found in the Illinois Rates sheet, the row should be able to properly process, ie words can separated by either a space ' ' or hyphen '-', so "License-Exempt Day Care Center" and "License Exempt Day Care Center" are both saved as license_exempt_day_care_home. It currently seems to be working for the file that was previously uploaded.

**Concerns**

There is a formatting issue with excel that both onboarding scripts don't seem to be configured for. Using the date format: MM/DD/YYYY does not get recognized by the ruby csv date parser and unfortunately upon opening up a csv file with excel, if not already configured, any date information will get automatically turned into this format. If you see a NotEnoughInfo error pop up from the onboarding script, this is probably the case. I believe downloading from google sheets directly as a csv does not run into this issue.

Some acceptable formats are "March 14, 2015" and "2015-03-14".

**Comment by rebeccakarasiktw** [comment]
Thanks for all the details @OliverCai0 
I just tried to upload the child but got this error: 


![Image](https://user-images.githubusercontent.com/26929739/190523258-ede88100-f856-4264-9e2e-99779fdcbace.png)



Is that because you already uploaded that kid in "Sheet 2"? If so, why can't I see them in the dashboard? 
Thanks!

**Comment by OliverCai0** [comment]
@rebeccakarasiktw yes I believe that is because the kid's name is already taken. If you log in with the credentials of mock Illinois user, the child shows up. I'm not sure why the child isn't showing up for admin accounts.

**Comment by OliverCai0** [comment]
I just made a modification to the visibility of the Illinois children. Admin IL users should be able to see all Illinois children.

**Comment by rebeccakarasiktw** [comment]
Hi @fabianraf I took a look and all of the kids were updated except the ones for IL Daycare 1. I assume it's something related to that user and we can discuss if that warrants another ticket to look into it. Moving this to done!
